### DISCUSSION TITLE
Can someone help me please :,)

### DISCUSSION THREAD

**Discussion by KikouDu18** [lucidrains/musiclm-pytorch#45] [Q&A]
Hi,
I'm a Pytorch beginner and I would like to install "musiclm-pytorch".
But when I try to install it with my terminal with this command : `pip install musiclm-pytorch`
And the terminal replies : `ERROR failed building wheel for fairseq`

I tried and I tried and I tried but there's always this error.
I have a great environment, I have the optimal pip and wheel versions (the most recent ones), I have `torchaudio`, `accelerate`, `beartype`, and so on...
Can someone please help me :,)

Python version : 3.9.0
Pytorch version : 2.0.0+cpu
Software used : Visual Studio 2022

Thank you beforehand :)

**Comment by mrfunkalot** [comment]
I had to install the Visual Studio Development Tools for C++ to get past the fairseq error I was getting.
